Update main documentation page (#92)
SUSE/saphanabootstrap-formula: README.md

You can find the latest development repositories at [SUSE's Open Build Service](https://build.opensuse.org/package/show/network:ha-clustering:sap-deployments:devel/saphanabootstrap-formula).

### Manual clone

```
git clone https://github.com/SUSE/saphanabootstrap-formula
cp -R cluster /srv/salt
```

**Important!** The formulas depends on [salt-shaptools](https://github.com/SUSE/salt-shaptools) package, so make sure it is installed properly if you follow the manual installation (the package can be installed as a RPM package too).

## Usage

To use the formula the `hana` entry must be included in the salt execution `top.sls` file. Here an example to execute the hana formula in all of the nodes:

```
# This file is /srv/salt/top.sls
base:
  '*':
    - hana
```

To configure the execution a pillar file is needed. Here an example of a pillar file for this formula: [pillar.example](https://github.com/SUSE/saphanabootstrap-formula/blob/master/pillar.example)
This file must be stored in `/srv/pillar` as `hana.sls` and the same folder must contain a `top.sls` to use it. For example:

```
# This file is /srv/pillar/top.sls
base:
  '*':
    - hana
```

> **Important!** The hostnames and minion names of the hana nodes must match the output of the `hostname` command.

Reviewer: ```suggestion
**Important!** The hostnames and minion names of the HANA nodes must match the output of the `hostname` command.
```

- If a masterless approach is used (as in the current automated deployment) the gpg private key must be imported in all the nodes. This might require the copy/paste of the keys.

## OBS Packaging

> The CI will automatically publish new releases to SUSE's Open Build Service every time a pull request is merged in the `master` branch. For that, update the new package version in [saphanabootstrap-formula.spec](https://github.com/SUSE/saphanabootstrap-formula/blob/master/saphanabootstrap-formula.spec) and

Reviewer: ```suggestion
The CI automatically publishes new releases to SUSE's Open Build Service every time a pull request is merged in the `master` branch. For that, update the new package version in [saphanabootstrap-formula.spec](https://github.com/SUSE/saphanabootstrap-formula/blob/master/saphanabootstrap-formula.spec) and
```

add the new changes in [saphanabootstrap-formula.changes](https://github.com/SUSE/saphanabootstrap-formula/blob/master/saphanabootstrap-formula.changes).

> The new version will published at:

Reviewer: ```suggestion
The new version is published at:
```
